#!/usr/bin/env python
PACKAGE = "mir_teleop"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("arm_cartesian_reference_link", str_t, 0, "Reference link which will be used by the arm Cartesian controller", "/base_link")

exit(gen.generate(PACKAGE, "mir_teleop", "TeleopJoypad"))
